docs/builds/guides/integration/installing-plugins.md

```diff
@@ -24,6 +24,14 @@ In order to start developing CKEditor 5 you will require:
 * [Node.js](https://nodejs.org/en/) 12.0.0+
 * npm 5.7.1+ (**note:** some npm 5+ versions were known to cause [problems](https://github.com/npm/npm/issues/16991), especially with deduplicating packages; upgrade npm when in doubt)
 
+<info-box warning>
+	When installing CKEditor 5 Framework packages, you need to make sure their versions match the version of the base editor package. If you use the latest version of, for example, `@ckeditor/ckeditor5-editor-classic` with an outdated version of `@ckeditor/ckeditor5-image`, npm or yarn will need to install two different versions of `@ckeditor/ckeditor5-core` because `@ckeditor/ckeditor5-editor-classic` and `@ckeditor/ckeditor5-image` may require different versions of the core package. This will result in [`ckeditor-duplicated-modules error`](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/support/error-codes.html#error-ckeditor-duplicated-modules).
```

Comment:
I'm not sure about this. The first sentence looks good, however, after reading the rest with multiple packages names it becomes tough. I think we should provide clear (and short) information that if you install a new package, it should be installed at a version that matches others. Details about why it happens could be found in the duplicated modules error docs. WDYT?

Reply:
Example:

> When installing CKEditor 5 Framework packages, you need to make sure their versions match the version of the base editor package. For example - if you'd like to install `@ckeditor/ckeditor5-alignment` package and your other packages are outdated, e.g. at version 18.0.0, you should consider updating your editor to the latest version or install the Alignment package at version 18.0.0.

Reply:
Well, this description was taken from duplicated modules error docs, so if it's tough here, it will be of no use in the error docs as well. So maybe we should also consider updating it there in the near future?
  
For me personally it was understandable, but of course we can rephrase it like in the example.

Reply:
I didn't mean it's not understandable, but it could be simplified. I think that simpler and shorter == better for anyone who meets the editor for the first time. As it comes to the duplicated modules error description - well, I guess it's a story for another episode :D